vowels.c

```diff
@@ -0,0 +1,65 @@
+/*Ashank Kumar, takes the vowels out of a string and prints the new string*/
+#include <stdio.h>
+#include <string.h>
+
+int main()
```

Comment:
Your code prints extra characters after every output. It works, but has these extra characters you didn't clean up.